.github/workflows/android_deploy_production_to_playstore.yml

```
@@ -47,6 +47,14 @@ jobs:
         run: |
           echo $ENV_PRODUCTION > .env
 
+      - name: Set up release signing configs
+        env:
+          ANDROID_RELEASE_KEYSTORE_BASE64: ${{ secrets.ANDROID_RELEASE_KEYSTORE_BASE64 }}
+          ANDROID_SIGNING_PROPERTIES_BASE64: ${{ secrets.ANDROID_SIGNING_PROPERTIES_BASE64 }}
+        run: |
+          echo $ANDROID_RELEASE_KEYSTORE_BASE64 | base64 --decode > android/config/release.keystore
+          echo $ANDROID_SIGNING_PROPERTIES_BASE64 | base64 --decode > android/signing.properties
```

@luongvo Since the `signing.properties` has line-break, and the default of `echo` command does not support adding it. Therefore, I suggest adding `-e` param
```suggestion
          echo -e $ANDROID_SIGNING_PROPERTIES_BASE64 | base64 --decode > android/signing.properties
```

https://stackoverflow.com/questions/70654298/how-to-properly-render-breakline-with-echo-command

---

FYI: We've tested it with easyHotel Project, even when creating the key-value pair at the secret:
```
key1=1
key2=2
```
or

```
key1=1\nkey2=2
```

After calling `echo`, the result of the file would be: 
```
key1=1key2=2
```

We can apply this for the env file too (in case having multiple line)

---

@doannimble Nope, it's because we using `echo $ENV_PRODUCTION > .env` without base64. I already aware of that issue with env file. 

Using `echo $ENV_PRODUCTION | base64 --decode > .env` would fix the issue here. I will fix it in https://github.com/nimblehq/flutter-templates/issues/226

---

or using `-e` would work too.

---

@doannimble Updated comment ☝️

---

I'm okay with this. It would be great if we can do the test when decoding the `signing` file and write it correctly by `echo`. 👌

---

@doannimble let me update env and signing.properties to use `echo -e`